config/default.yml

```diff
@@ -0,0 +1,58 @@
+AvoidOutlineForSingleExample:
+    Enabled: true
+AvoidPeriod:
+    Enabled: true
+AvoidScripting:
+    Enabled: true
+BackgroundDoesMoreThanSetup:
+    Enabled: true
+BackgroundRequiresMultipleScenarios:
+    Enabled: true
+BadScenarioName:
+    Enabled: true
+BeDeclarative:
+    Enabled: true
+FileNameDiffersFeatureName:
+    Enabled: true
+MissingExampleName:
+    Enabled: true
+MissingFeatureDescription:
+    Enabled: true
+MissingFeatureName:
+    Enabled: true
+MissingScenarioName:
+    Enabled: true
+MissingTestAction:
+    Enabled: true
+MissingVerification:
+    Enabled: true
+InvalidFileName:
+    Enabled: true
+InvalidStepFlow:
+    Enabled: true
+RequiredTagsStartsWith:
+    Enabled: false
```

Comment:
if it is a regexp, it could be enabled everytime. so just set it to match everything as Default.